### ISSUE TITLE
test_mask_feature CI failure

### ISSUE THREAD

**Issue by puririshi98** [pyg-team/pytorch_geometric#5670]
### 🐛 Describe the bug

```
======================================================= FAILURES =======================================================
__________________________________________________ test_mask_feature ___________________________________________________

    def test_mask_feature():
        x = torch.tensor([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]],
                         dtype=torch.float)

        out = mask_feature(x, training=False)
        assert out[0].tolist() == x.tolist()
        assert torch.all(out[1])

        torch.manual_seed(4)
        out = mask_feature(x)
        assert out[0].tolist() == [[1.0, 2.0, 0.0, 0.0], [5.0, 6.0, 0.0, 0.0],
                                   [9.0, 10.0, 0.0, 0.0]]
        assert out[1].tolist() == [[True, True, False, False]]

        torch.manual_seed(5)
        out = mask_feature(x, mode='row')
        assert out[0].tolist() == [[1.0, 2.0, 3.0, 4.0], [0.0, 0.0, 0.0, 0.0],
                                   [9.0, 10.0, 11.0, 12.0]]
        assert out[1].tolist() == [[True], [False], [True]]

        torch.manual_seed(7)
        out = mask_feature(x, mode='all')
>       assert out[0].tolist() == [[1.0, 0.0, 0.0, 4.0], [5.0, 6.0, 7.0, 0.0],
                                   [9.0, 10.0, 0.0, 12.0]]
E       assert [[1.0, 2.0, 0..., 11.0, 12.0]] == [[1.0, 0.0, 0...0, 0.0, 12.0]]
E         At index 0 diff: [1.0, 2.0, 0.0, 0.0] != [1.0, 0.0, 0.0, 4.0]
E         Use -v to get the full diff

test/utils/test_augmentation.py:57: AssertionError
```

### Environment

latest pyg and pyg-lib

**Comment by puririshi98** [comment]
```
>>> torch.manual_seed(7)
<torch._C.Generator object at 0x7f55d59652b0>
>>> mask_feature(x, mode='all')
(tensor([[ 1,  2,  0,  0],
        [ 0,  0,  7,  0],
        [ 0,  0, 11, 12]]), tensor([[ True,  True, False, False],
        [False, False,  True, False],
        [False, False,  True,  True]]))
```
not sure if theres really an error or the ground truth value jsut needs to be updated. it all depends on the seed, not sure why things would have changed

**Comment by EdisonLeeeee** [comment]
This happens when using `mask = x.bernoulli(1 - p)`, which would return different results on different OS/machines even if the seed is fixed. I'll make a PR to address this soon.
